src/event_form/steps/event-form-step3.controller.js

```diff
@@ -226,87 +225,68 @@ function EventFormStep3Controller(
     $scope.selectedLocation = false;
     $scope.locationAutocompleteTextField = '';
     $scope.locationsSearched = false;
+    $scope.selectedCityObj = city;
 
     controller.stepUncompleted();
-    controller.getLocations(city);
   }
 
-  /**
-   * Get locations for Event.
-   * @returns {undefined}
-   * @param {Object} city
-   */
-  controller.getLocations = function (city) {
+  controller.getPlaces = function(filterValue) {
 
-    function showErrorAndReturnEmptyList () {
-      $scope.locationAutoCompleteError = true;
-      return [];
+    // exit code when the filterValue has < 3 characters
```

Comment:
Nitpicking, but `exit code` reminds me of the status code that CLI commands return after running. Could we change it to something more descriptive like `Do not look for place suggestions until the user has entered at least 3 characters`?